apps/vtex-my-subscriptions-3/react/components/DetailsPage/History/HistoryItem.tsx

```diff
@@ -13,18 +15,29 @@ const HistoryItem: FunctionComponent<Props> = ({
 
   switch (status) {
     case 'SUCCESS':
+      break
```

Comment:
I don't have much context about the app, but adding the break could not change the behavior. In that case, maybe ignoring the rule was a safer path